db/schema.rb

```diff
@@ -689,8 +689,8 @@
     t.string "category_of_law", null: false
     t.string "category_law_code", null: false
     t.string "ccms_matter_code"
-    t.string "client_involvement_type_ccms_code", null: false
-    t.string "client_involvement_type_description", null: false
+    t.string "client_involvement_type_ccms_code"
+    t.string "client_involvement_type_description"
```

Comment:
It looks like these fields are nullable, but I didn't remove the constraint.

Has anyone run in to this before?